types/node/node-tests.ts

```diff
@@ -35,6 +35,11 @@ import Module = require("module");
 // Specifically test buffer module regression.
 import { Buffer as ImportedBuffer, SlowBuffer as ImportedSlowBuffer } from "buffer";
 
+// Forward-declare SharedArrayBuffer (available starting es2017)
+interface SharedArrayBuffer {
```

Comment:
This shouldn't be needed here as the forward declaration is already in index.d.ts.

Reply:
Done.